pkg/query/endpointset.go

```diff
@@ -183,16 +185,21 @@ type endpointSetNodeCollector struct {
 	storePerExtLset map[string]int
 
 	connectionsDesc *prometheus.Desc
+	requiredLabels  []string
 }
 
-func newEndpointSetNodeCollector() *endpointSetNodeCollector {
+func newEndpointSetNodeCollector(requiredLabels ...string) *endpointSetNodeCollector {
```

Comment:
```suggestion
func newEndpointSetNodeCollector(labels ...string) *endpointSetNodeCollector {
```

Reply:
Why `required` word? Is there any confusion that without `required` users/readers of this function might not understand something?

Reply:
maybe required not needed. will remove